vllm/engine/arg_utils.py

```diff
@@ -293,7 +293,7 @@ def add_cli_args(parser: FlexibleArgumentParser) -> FlexibleArgumentParser:
         parser.add_argument('--block-size',
                             type=int,
                             default=EngineArgs.block_size,
-                            choices=[8, 16, 32],
+                            choices=[8, 16, 32, 128, 256, 512, 1024, 2048],
```

Comment:
I am curious why we skip 64? Is it for a specific reason? Thanks

Reply:
This may have been an oversight during testing, but it's been addressed by #7562